spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+describe User do
+
+ let!(:user) do
+   User.create(email: 'test@test.com', password: 'secret1234',
+               password_confirmation: 'secret1234')
+ end
+
+ it 'authenticates when given a valid email address and password' do
+   authenticated_user = User.authenticate(user.email, user.password)
+   expect(authenticated_user).to eq user
+ end
+
+ it 'does not authenticate when given an incorrect password' do
+    expect(User.authenticate(user.email, 'wrong_stupid_password')).to be_nil
+  end
```

Comment:
Lint/BlockAlignment: `end` at 15, 2 is not aligned with `it 'does not authenticate when given an incorrect password' do` at 13, 1